java/src/main/java/genericpubsub/ManagedSubscribe.java

```diff
@@ -38,14 +34,14 @@ public class ManagedSubscribe extends CommonContext implements StreamObserver<Ma
     private StreamObserver<ManagedFetchRequest> serverStream;
     private Map<String, Schema> schemaCache = new ConcurrentHashMap<>();
     private final CountDownLatch serverOnCompletedLatch = new CountDownLatch(1);
-    private boolean isActive;
-    private int receivedEvents = 0;
+    public static AtomicBoolean isActive = new AtomicBoolean(false);
```

Comment:
These changes (as well as some of the other changes in this file) are intended to standardize this ManagedSubscribe example with the normal [Subscribe](https://github.com/forcedotcom/pub-sub-api/blob/main/java/src/main/java/genericpubsub/Subscribe.java) example